Diagrams/UseCase/HotelManager.drawio

```diff
@@ -1 +1 @@
\ No newline at end of file
+<mxfile host="Electron" modified="2022-10-08T02:50:28.342Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/20.2.3 Chrome/102.0.5005.167 Electron/19.0.11 Safari/537.36" etag="LUxin0SA_UWRJFb31L1Z" version="20.2.3" type="device"><diagram id="VzB0pJPlA92tWoaxJZA9" name="Page-1">7Vxbc5s4GP01mX0yg+7iMZum7ey000wznW2fOsQoNhtsPFhO7P31K2yBkYRtgm1wuvVLkBACjo6+O7lCN5PlhyycjT+nkUiuoB8tr9C7KwgB8KH6k/esdA9nZNMzyuJI92077uN/he70de8ijsTcGCjTNJHxzOwcptOpGEqjL8yy9MUc9pgm5l1n4Ug4HffDMHF7/44jOS5fzN+e+Cji0VjfmhN9YhIWg3XHfBxG6UulC91eoZssTeXmaLK8EUmOXoHL5rr3O86WD5aJqWxywZfFp8H7u78/Jj/Fl9Xt8DteLr4NoF6f5zBZ6DfWTytXBQRZuphGIp/Fv0J/voxjKe5n4TA/+6JWXfWN5SRRLaAO3afSD/osMimWlS79lB9EOhEyW6kh+iykGjHNmQFiuuNluwKkIMi4Aj6AxchQL/uonH2LjDrQ4LwCKODg9DGVa7Lfy/Dx0QFNrfYsP1xMkuuhTDMFTY5ArJj1KXwQyV06j2WcTtWQh1TKdFIZcJ3Eo/yETC1w04VM4qm4Kbnunwbxks0acRy4gKMavOm50HZZeb2QY/WCCp41ajbcak4lEUQn/KTERKuEpoIWgDVw8XPBhR24bsZi+JSLxfyviAaxgsxXsmYy7xU6ZEFHcd/QkZ3QZWIusmcl9y4RuBKQ3oCjDnC30xyr4WKupJnI/piv0QpXF4Ubq1El3eLGHNy+boiW30xzrVfIMDQh651p3EHsXqyp9hDnF/cIFeEWu3rHKnDFWTgdru2UjTzrX30iy9ggsG/QgGvcuQhNo+vcnVCtaToVJioKjGz1Xdtj68aPvOGRovluWT35bnXIdJuni2woDpujMsxGQh5WbyIyvBx3baqWdQ30RV8mEkWgZ9M3qlsPfYe7NFZvtnO/EN9a0s1766uqvow1EUXmRJhYE22AcSZa06N87SMY08Bt2s+YZSy/V45/VI63ZMkbq0rjTmSxenSRVdl0kbzDv3l3Ft6hGt7RcJJL6USuV7baEku51pS6b2SM0K3drE1nYlrPWs2wZrwtOAoMjh5gaBTOx+uQw+aq6H2u6o8lL2tIXkh/TfY6ToQ90bnZ6/qpnbK3lJZbAfmjeq5LLgZvUoFjwjzCEAkQgJz7rNgpRXQXcQ9yDBmAmGEKKGtJVCsCWEaDuyKqGxXoX8yWknW/mL0EctO3aSVYrKM+8HzcjsGYQM9HwfZnbhSCmDP1uTntBmz2cTqeDpNFJM5B6oAZtPYo53upvYeil2EUN6U7vyi6Y2DSndn+dVOuE2xOxFHH0toNqXXK7H38bCmV23MRN+QivCwuYmaKXkZai17CLDFu5wR30FEtcB6sLofN8gHz3c/MrDgW5Ub+VR1sZjwt191gaC9SvBq32G+YNNkdA9/z1S42xfVGK+zZJOuWLf3b7xzye+e03DmnIjd0oxtlkn01l8LNi0hlepsEncssfRI3aZKn3IvA26PinNUV6hz7UPFEVLPzRfJ9EkdRsisibpZE7OTcK2oeLOgHpQtUoVFQQyPo72bMcSl4114sMqPhcxgn4UOSp6wuIDVq5atATfFCt/kEWGeQ7JayHeQT2oa8mkaE+5J+wHLZkL0dGkcc/MCreGssIOa8Hcd54YWoeeWs0cI5084aPVrbm6Q+pORrPLz2Oh42ddUuTMkD6OG8WIZyBkBAAov0wGOAMMgZgQRR1G4LUMI8NXX548ZN1OS1AY3Tb4jBF/Fz/lf0+Wn54evs2/0yG4+fBq4ANVVwmslxOkqnYfIpzavm1vz7R0i50iWl4UKmtpC1JXBd+MsRiAdoV/v0r5PCLssa06ep2qp9SuTKCPXy97qpMfoVYSd9ol54dL807LXjgl5hD/6vsPNeYXfsmhtdL+gsR9cV1I7HUbMMO50QYBUGgBr37VwV1LVAu6nWz2kUP25W+mSFYEdhhq10NOmyjroWtNelSE6d9juYijuJmGgaIN6hFLsxeBELPE5RgAKIuLJviZ0+U74wxhAjH1MGUMs0s13pis4X79otiQ263a0j0Keoaz1uZ/p2NKrLGs3dxkJtNGocKwWSHV1mfhRgKLAAA11WAtdr2hpVa3zZoBXAZeLXuyqoqfAs4Nt81HARuFFs867L7xvqgeu3yKsbJQqO9HE6ChtZuwrY+q2xokRG/AkybMwLgRGA4siKQJ1ZjfZcrdUk5Ak83y/TmSfIb56GxPBNkJgxL4ABz2P7BFM/QJbECzyKA6AcLsiUOah+LTkeQM9nPmCUBZgohWTmVQeB7yHic44CBmlASLfxz76rt1rUuByK7J+Qydwl8u6ofn+ZeuoplpY/YMpQAj3uV87ydjSmzKzQRdZdDEnNuOU5nZvFb6JSC3JkimqfogNcPreoblot3i/ByV6CU5PgsCW/+V5+0175XRja1U9Vw/WXvabLNRyH05E4umziKNeBWNE31OWn+PXouS7XpUiHE27lmvjbHi71tpfpXosf+ZbF33Iz5ylnM7TXd575dR/YXT4Hm358dIEcVEa4xylRioNxwDA2CxIIhl7+yYbvM4Apb2v2qwk8gDBDDAKuhCI0boIh8xjO6Qg5Qeo+HbPxdZ5tX/YStUp3O7WXmn7peYkEx3sJTk5DcLCX4OhMBFfN7b822wzf/oc4dPsf</diagram></mxfile>
```

Comment:
Can you move your changes to a new .drawio file? Since there is HotelManager and Customer perspectives of the app, we should separate them out. A suggestion is Customer.drawio.

Also, can you include a screenshot of your use-case in the PR description?

Reply:
Ok